Src/libCZI/SingleChannelScalingTileAccessor.cpp

```diff
@@ -78,8 +78,9 @@ CSingleChannelScalingTileAccessor::CSingleChannelScalingTileAccessor(const std::
     return IntSize{ static_cast<uint32_t>(roi.w * zoom),static_cast<uint32_t>(roi.h * zoom) };
 }
 
-void CSingleChannelScalingTileAccessor::ScaleBlt(libCZI::IBitmapData* bmDest, float zoom, const libCZI::IntRect& roi, const SbInfo& sbInfo)
+void CSingleChannelScalingTileAccessor::ScaleBlt(libCZI::IBitmapData* bmDest, float zoom, const libCZI::IntRect& roi, const SbInfo& sbInfo, const libCZI::ISingleChannelScalingTileAccessor::Options& options)
```

Comment:
I see your point, however - I'd choose to go with passing the options-object. We actually need two parameters from the options-object currently, and it is not unlikely that more options will be needed in the future. Since this is a private method, I'd consider it as "to be expected" that only the "relevant options" are used. This makes the method for sure less versatile and could be a problem when re-using it in some other context, but I'd argue that refactoring has to wait until this is the case.